CHANGELOG.md

```diff
@@ -1,14 +1,24 @@
 # Changelog
-* Friday 20th May 2022
-v0.16.2
+
+- Friday \*\* June 2022
+  v0.16.2
+
+## Enhancements
+
+- We introduced a new function called `set_assume_org_id` for incoming super organization feature. By using this method, you can assume the id of a child organization as their super organization, and make your calls on behalf of them.
```

Comment:
I think here we need to explain what a superorganisation is.

Reply:
agreed